itests/org.jboss.tools.cdk.ui.bot.test/README.md

```diff
@@ -0,0 +1,30 @@
+- downloaded CDK 2.x vagrant file
+- downloaded and configured CDK 3.1 binary, `minishift setup-cdk` must be called before test is run in IDE
+- TODO: downloaded and configured CDK 3.2 binary, and have called setup-cdk on it (which creates ~/.minishift folder, and prepares cdk to be able to start from IDE)
+3. For successful registration of vm-guest rhel image your account must have signed Terms and Conditions (https://developers.redhat.com/terms-and-conditions/).
+4. For ease of configuration and automation there is a possibility to use cdk configuration scripts (https://github.com/odockal/cdk-scripts)
+
+#### Running CDK integration tests from IDE
+There are three suites for CDK integration tests (full suite (CDKAllTestsSuite), smoke suite (CDKSmokeTestsSuite)) and CDK 3.x only test suite (CDK3AllTestsSuite). To run CDK integration tests from IDE, perform following steps:
+1. Install RedDeer to your IDE from update site: http://download.eclipse.org/reddeer/releases/latest (RedDeer is on DevStudio TP, RedDeer project repo URL: https://github.com/eclipse/reddeer)
+2. Select desired java suite (e.g. CDKAllTestsSuite, CDKSmokeTestsSuite, CDK3AllTestsSuite) and in its context menu select _Run As_ - _Run Configurations..._
+3. In Run Configurations dialog double click on RedDeer Test and a new RedDeer test run configuration for your suite will be created
+4. Select tab Argument and fill in following properties **with credentials** to VM arguments:
+
+`-Dminishift.hypervisor=kvm/virtualbox/hyperv/xhyve -Dminishift.path=/path/to/your/minishift/binary -Ddevelopers.username=yourusername -Ddevelopers.password=password -DskipTests=false -Dusage_reporting_enabled=false -Dvagrantfile.path=/path/to/your/vagrantfile/folder/`.
+5. Confirm changes and run test suite.
+
+#### Running CDK integration tests from command line through maven
+For execution of CDK integration tests from command line you will need maven. At first build _org.jboss.tools.cdk.reddeer_ plugin with maven (`mvn clean install`). Then run in _org.jboss.tools.cdk.ui.bot.test_ plugin following command **with filled credentials** with basic authentication. Default test suite is CDKAllTestsSuite. User can switch to different suite class by using profiles: `-Psmoke-tests` (CDKSmokeTestsSuite), `-Pall-tests` (CDKAllTestsSuite) or `-Pcdk3-all-tests` (CDK3AllTestsSuite).
+To run tests, type this command:
+
+`mvn clean verify -Dtest.installBase=/path/to/ide/to/run/against -Dminishift.hypervisor=kvm/virtualbox/hyperv/xhyve -Dminishift.path=/path/to/your/minishift/binary -Ddevelopers.username=yourusername -Ddevelopers.password=password -DskipTests=false -Dusage_reporting_enabled=false -Dvagrantfile.path=/path/to/your/vagrantfile`.
+
+You can also run a single test class. To utilize all advantages of RedDeer suite, be sure your test class is annotated with RunWith(RedDeerSuite.class) annotation. If you want to run a single test class instead of a test suite, replace argument -Dscope by argument -Dtest. 
```

Comment:
```-Dtest= ``` didnt work for me in Eclipse. Even though I added this to the cmd-line arguments the whole suite (that is indicated in the Main tab?) is being run.

Reply:
To be honest, I just copy-pasted this section from openshift itests. I will test this myself. Thanks

Reply:
I see it now. I did not use @RunWith(RedDeerSuite.class) annotation on single classes. To be repaired.